File: sixthSeminar/src/main/java/org/sopt/sixthSeminar/repository/PostRepository.java
package org.sopt.sixthSeminar.repository;

import org.sopt.sixthSeminar.domain.Post;
import org.sopt.sixthSeminar.domain.ServiceMember;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface PostRepository extends JpaRepository<Post, Long> {
    List<Post> findAllByServiceMember(ServiceMember serviceMember);
}

File: secondKotlin/src/main/kotlin/org/sopt/secondKotlin/repository/PostRepository.kt
package org.sopt.secondKotlin.repository

import org.sopt.secondKotlin.domain.Member
import org.sopt.secondKotlin.domain.Post
import org.springframework.data.jpa.repository.JpaRepository

interface PostRepository : JpaRepository<Post, Long> {
    fun findAllByMember(member: Member) : List<Post>
}